x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rule_snooze/panel/base_snooze_panel.tsx

```diff
@@ -142,6 +144,114 @@ export const BaseSnoozePanel: React.FunctionComponent<BaseSnoozePanelProps> = ({
     [scheduledSnoozes]
   );
 
+  const renderSchedule = () => {
+    if (!showAddSchedule) {
+      return null;
+    }
+    return (
+      <>
+        <EuiHorizontalRule margin="s" />
+        {!hasSchedules && (
+          <>
+            <EuiFlexGroup>
+              <EuiFlexItem grow>
+                <EuiButton
+                  fill
+                  color="primary"
+                  onClick={onClickAddSchedule}
+                  data-test-subj="ruleAddSchedule"
+                  iconType="calendar"
+                >
+                  {i18n.translate('xpack.triggersActionsUI.sections.rulesList.addSchedule', {
+                    defaultMessage: 'Add schedule',
+                  })}
+                </EuiButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </>
+        )}
+        {hasSchedules && (
+          <>
+            <EuiFlexGroup alignItems="center">
+              <EuiFlexItem>
+                <EuiTitle size="xxxs">
+                  <h5>
+                    {i18n.translate(
+                      'xpack.triggersActionsUI.sections.rulesList.snoozeSchedulesTitle',
+                      {
+                        defaultMessage: 'Schedules',
+                      }
+                    )}
+                  </h5>
+                </EuiTitle>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmpty
+                  color="danger"
+                  size="xs"
+                  onClick={() => setIsRemoveAllModalVisible(true)}
+                  data-test-subj="ruleRemoveAllSchedules"
+                >
+                  {i18n.translate('xpack.triggersActionsUI.sections.rulesList.removeAllButton', {
+                    defaultMessage: 'Remove all',
+                  })}
+                </EuiButtonEmpty>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+            <EuiFlexGroup direction="column" gutterSize="xs" data-test-subj="ruleSchedulesList">
+              {scheduledSnoozes!
+                .filter((s) => s.id)
+                .map((schedule) => {
+                  const isActive = schedule.id && activeSnoozes.includes(schedule.id);
+                  return (
+                    <EuiFlexItem key={`snooze-${schedule.id}`}>
+                      <button
+                        style={{
+                          paddingLeft: '9px',
+                          paddingRight: '9px',
+                          // Replicate euiPanel--accent vs euiPanel--subdued
+                          // Applying these classNames by themselves doesn't work due to a CSS-in-JS issue with EuiPanel
+                          color: isActive ? '#a8376a' : euiTheme.colors.subduedText,
+                          backgroundColor: isActive ? 'rgba(240,78,152,0.2)' : euiTheme.colors.body,
+                        }}
```

Comment:
can we use `import { css } from '@emotion/react';` and is it possible to avoid `'#a8376a'` and `rgba(240,78,152,0.2)`, that's not part of `euiTheme`

Reply:
I'm not sure what color this this, since I didn't create this component, I simply moved it. Maybe we can ask design